### PR TITLE
Adding example social media posts --cshort & --kaslin

### PR DESCRIPTION
**Pull request by chris-short**
Adding example social media guidelines (and some markdown listing because I'm THAT guy).

### CODE REVIEW

File: communication/marketing-team/social-guidelines.md [hunk 1]
```diff
@@ -31,15 +32,33 @@ Kubernetes Organization Members with access to account are:
 - Significant historical events as related to the project (Kubernetes birthday, previous events, etc.)
 - Steering Committee Announcements
 
+### Example Social Media
+
+- Weekly on Sunday: Link with community calendar and "This week in upstream community group meetings..."
+- As they come in:
+  - Emails from k-dev
+  - #Shoutouts (as requested; please include social handles)
```
Comment (parispittman) [inline communication/marketing-team/social-guidelines.md:40]: This should be pulled from the shoutout channels in slack either daily or every two days, etc some kind of cadence. Might want to include that in here to be descriptive for future folks.

File: communication/marketing-team/social-guidelines.md [hunk 2]
```diff
@@ -31,15 +32,33 @@ Kubernetes Organization Members with access to account are:
...
+  - Requests from issue templates
+- Scheduled launches
+  - Your upcoming blog posts
```
Comment (parispittman) [inline communication/marketing-team/social-guidelines.md:43]:
```
Contributor blog posts from storytellers
```

File: communication/marketing-team/social-guidelines.md (hunk 1 above)
Comment (parispittman) [inline communication/marketing-team/social-guidelines.md:40]:
```
#Shoutouts (from #shoutouts slack channel; try to include social handles or github handles if possible)
```

File: communication/marketing-team/social-guidelines.md [hunk 3]
```diff
@@ -31,15 +32,33 @@ Kubernetes Organization Members with access to account are:
 - Significant historical events as related to the project (Kubernetes birthday, previous events, etc.)
 - Steering Committee Announcements
 
+### Example Social Media
```
Comment (parispittman) [inline communication/marketing-team/social-guidelines.md:35]:
```
Content and Suggested Schedule
```

File: communication/marketing-team/social-guidelines.md [hunk 4]
```diff
@@ -31,15 +32,33 @@ Kubernetes Organization Members with access to account are:
...
+  - Emails from k-dev
+  - #Shoutouts (as requested; please include social handles)
+  - Requests from issue templates
+- Scheduled launches
+  - Your upcoming blog posts
+  - Requests from issue templates
+- Every Wednesday (example)
+  - New contributor Wednesday - tweet a bunch of good-first-issue label GitHub issues (TODO: Automate)
+- Event support:
+  - (Based on issue template)
```
Comment (parispittman) [inline communication/marketing-team/social-guidelines.md:48]: Info gathered from a filed issue template

**Comment by parispittman** [comment]
Added my comments. I have some more ideas to add but we can push this and iterate. Thank you!

File: communication/marketing-team/social-guidelines.md (hunk 1 above)
Comment (mbbroberg) [inline communication/marketing-team/social-guidelines.md:40]:
```suggestion
  - #Shoutouts (from #shoutouts slack channel; try to include social handles or github handles if possible)
```

File: communication/marketing-team/social-guidelines.md (hunk 4 above)
Comment (mbbroberg) [inline communication/marketing-team/social-guidelines.md:48]:
```suggestion
  -  (with information from the issue template)
```

**Pull request merged**